### PR TITLE
Update bridge-lpt.md

### PR DESCRIPTION
**Pull request by 0xVires**
update the L2 -> L1 guide

### CODE REVIEW

**Comment by hthillman** [comment]
@0xVires looks like the build is failing - can you try running `yarn build` and see if you get errors? usually this is because of a broken link

**Comment by 0xVires** [comment]
did that work? I'm still a git noob ;)

**Review by hthillman — APPROVED** [review]

**Comment by hthillman** [comment]
looks good to me!

File: docs/delegators/how-to-guides/bridge-lpt.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ Please note that you will need some ETH in your wallet to complete this guide. I
...
     - We are looking into options to remove this step. If it seems complex, we recommend swapping to ETH via [Uniswap](https://app.uniswap.org/#/swap?chain=arbitrum) and then transferring the ETH from L2 to L1 instead.
-- Enter the L2 LPT contract address ([0x289ba1701C2F088cf0faf8B3705246331cB8A839](/protocol/reference/deployed)) as the destination address on the [Arbitrum Bridge](https://bridge.arbitrum.io/)
+- Next, click on the arrow below the amount input. The button will switch from "Deposit" to "Withdraw"
+- Click on "Token" -> "View Token Lists" -> enable "Arbed CMC List" and click on "View Tokens" again. You should now be able to choose LPT in the list.
+- After you sent the withdraw tx, you need to wait ~1 week until you can claim your LPT on L1. You'll see the countdown below. Once the confirmation period is over, you can click "Claim" (you need to be connected to mainnet for that).
```
Comment (hthillman) [inline docs/delegators/how-to-guides/bridge-lpt.md:39]: typo: After you se**n**d

also a phrasing update:
```
you can connect to the Ethereum mainnet and click "Claim" to access your L1 LPT.
```

**Review by hthillman — CHANGES_REQUESTED** [review]
actually one minor change

File: docs/delegators/how-to-guides/bridge-lpt.md (hunk 1 above)
Comment (0xVires) [reply docs/delegators/how-to-guides/bridge-lpt.md:39]: sent seems correct to me - was meant to be in past tense. 
Agree on the phrasing change, maybe even:  `you can connect to the Ethereum mainnet and click "Claim" to withdraw your LPT on L1.`

**Review by hthillman — APPROVED** [review]

**Pull request merged**